README.rst

```diff
@@ -1,14 +1,23 @@
 PHP Coding Standards Fixer
 ==========================
 
-The PHP Coding Standards Fixer tool fixes *most* issues in your code when you
-want to follow the PHP coding standards as defined in the PSR-1 and PSR-2
-documents and many more.
+The PHP Coding Standards Fixer (PHP CS Fixer) tool fixes your code to standards;
+whether you want to follow PHP coding standards as defined in the PSR-1, PSR-2, etc.,
+or other community driven ones like the Symfony one.
+You can define your (teams) style through configuration.
+
+Besides code style fixes it can (micro) optimize and modernize your code.
```

Comment:
I would revert possibilities, putting `modernize` on front to put focus on it,
giving example would also be nice (like upgrade syntax to newer PHP engine)